### ISSUE TITLE
dependencies.sh is not a real dependencies file.

### ISSUE THREAD

**Issue by Lvl4Sword** [saurabhmarpadge/extramagnet#3]
Your bash script is not the way to do dependencies. Look into something like https://github.com/habnabit/passacre and https://docs.python.org/3/install/index.html for help creating a proper installation.

**Comment by saurabhmarpadge** [comment]
Yes, your are right that would me a more proper way of doing it.
I thought it would be a bit tough, anyways I will give it a try.
